### ISSUE TITLE
Stackdriver logs not outputting all of jsonPayload

### ISSUE THREAD

**Issue by hershmire** [ovhemert/pino-stackdriver#2]
I'm trying to integrate this library into a project and although I'm seeing the logs show up with the proper levels in Stackdriver, I'm noticing that I'm missing most of the `jsonPayload` key/values that I would normally expect such as `hostname`, `pid`, and any other items I set up from Pino's `base` option. This is also true for any custom object I send for the log itself. 

```js
const stackdriver = require('pino-stackdriver');
const pino = require('pino');

const credentials = process.env.GOOGLE_APPLICATION_CREDENTIALS;
const projectId = 'xxxxx';
let writeStream;

if (credentials) {
  writeStream = stackdriver.createWriteStream({
    credentials,
    projectId,
  });
}
const logger = pino(
  {
    level: 'trace',
  },
  writeStream || process.stdout
);

const child = logger.child({ childProperty: 'property' });
child.info({ prop: 'something' }, 'PINO this is an info log from child 1!');
```

All I seeing in Stackdriver logs are:

```js
{
 insertId:  "..........44gwuhZIKBPA_UhWeVQvNxw"  
 jsonPayload: {
  message:  "PINO this is an info log from child 1!"   
 }
 logName:  "projects/xxxxx/logs/pino_log"  
 receiveTimestamp:  "2018-12-22T21:56:44.243763581Z"  
 resource: {
  labels: {
   project_id:  "xxxxx"    
  }
  type:  "global"   
 }
 severity:  "INFO"  
 timestamp:  "2018-12-22T21:56:41.496000051Z"  
}
```

Notice that `jsonPayload` is only showing a `message` key/value. What I'm unsure of is if this is a limitation with pino-stackdriver or a configuration issue on my end. 

When I view the Pino log output without writing to this stackdriver stream, I see all the key/values as expected. 

```json
{
  "level": 30,
  "time": 1545517082929,
  "msg": "PINO this is an info log from child 1!",
  "pid": 65250,
  "hostname": "xxxxx",
  "childProperty": "property",
  "prop": "something",
  "v": 1
}
```

When I try other logging solutions such as Bunyan with bunyan-stackdriver, all the necessary key/values get populated.

**Comment by ovhemert** [comment]
This issue was actually already on my todo list.

Just published v1.1.0 which adds all the Pino properties to the jsonPayload of the log entry.

Let me know if that works for you.

**Comment by hershmire** [comment]
@ovhemert v1.1.0 seems to work as expected. Thanks!
